stubs/inertia/resources/js/Pages/Auth/ResetPassword.vue

```diff
@@ -43,7 +43,9 @@
         },
 
         props: {
+            auth: Object,
             email: String,
+            errors: Object,
```

Comment:
Would it maybe be smart to always list the top level props (`auth` and `errors`) first? 🤔

Reply:
I was thinking about this but as all breeze components list props in alphabetical order, so I followed the same order. Maybe when developers create their own pages they may prefer to list (`auth `and `errors`) first.

Reply:
Yeah fair enough. 👍